raft/src/main/java/org/apache/kafka/raft/MetadataLogConfig.java

```diff
@@ -112,15 +108,15 @@ public class MetadataLogConfig {
      * @param deleteDelayMillis The amount of time to wait before deleting a file from the filesystem
      */
     public MetadataLogConfig(int logSegmentBytes,
-                             int logSegmentMinBytes,
+                             int internalLogSegmentMinBytes,
```

Comment:
Hmm, this should be renamed to internalLogSegmentBytes since it's no longer the minimum, right?